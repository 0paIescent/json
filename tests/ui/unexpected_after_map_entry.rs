#[macro_use]
extern crate serde_json;

fn main() {
    json!({ "k": true => });
}
