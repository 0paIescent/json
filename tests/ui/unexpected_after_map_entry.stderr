error: no rules expected the token `=>`
  --> $DIR/unexpected_after_map_entry.rs:13:23
   |
13 |     json!({ "k": true => });
   |                       ^^ no rules expected the token `=>`

error: aborting due to previous error

